spec/features/creating_peeps_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'Adding a new Peeps' do
+  scenario 'A user can add a peep to Chitter' do
+    visit('/peep')
+    fill_in('peep', with: 'test peep')
+    click_button('Submit')
+    expect(page).to have_content 'test peep'
```

Comment:
Nice simple test that is easy to read but is robust enough to test the feature properly